src/main/java/io/reactivex/Maybe.java

```diff
@@ -2213,7 +2213,7 @@ public final T blockingGet(T defaultValue) {
     @SchedulerSupport(SchedulerSupport.NONE)
     public final Maybe<T> defaultIfEmpty(T defaultItem) {
         ObjectHelper.requireNonNull(defaultItem, "item is null");
-        return switchIfEmpty(just(defaultItem));
+        return switchIfEmpty(Single.just(defaultItem));
```

Comment:
Why was this changed, the original `just` should still work?!

Reply:
Of course the original `just` still works. Just thought that it would be better to wrap the `T defaultItem` into `Single` because the `defaultItem` cannot be empty.

Reply:
Please restore the original state. The less change to existing and practically unrelated code regarding the PR the better.